ui/desktop/src/components/UserMessage.tsx

```diff
@@ -8,8 +8,8 @@ export default function UserMessage({ message }) {
   const urls = extractUrls(message.content, []);
 
   return (
-    <div className="flex justify-end mb-[16px]">
-      <div className="flex-col max-w-[90%]">
+    <div className="flex justify-end mb-[16px] w-full">
```

Comment:
Does this change make every user message the full width of the chat space?

Reply:
Solution would be `max-w-full` if so and we don't want that. This would let a single word message like "hi" have a small bubble, but constrain maximum width.

Reply:
no doesn't make it full width